GameServerLib/Logic/Logger.cs

```diff
@@ -74,10 +79,35 @@ private class LogWriter
         {
             public string _logFileName;
             private object _locker = new object();
+            private StringBuilder stringBuilder = new StringBuilder();
```

Comment:
Should be `_stringBuilder`
BTW, it shouldn't be. Calling it with what it does would be better.